sys/include/ztimer.h

```diff
@@ -503,11 +503,21 @@ static inline void ztimer_init_extend(ztimer_clock_t *clock)
 }
 
 /* default ztimer virtual devices */
+/**
```

Comment:
just a suggestion: when reading this, would it make sense to have ZTIMER_USEC and ZTIMER_MSEC both on top and the BASE below?

Reply:
Do you mean:

A)
```
/**
 * @brief   Default ztimer microsecond clock
 */
extern ztimer_clock_t *const ZTIMER_USEC;

/* default ztimer virtual devices */
/**
 * @brief   Base ztimer for the microsecond clock
 */
extern ztimer_clock_t *const ZTIMER_USEC_BASE;

/**
 * @brief   Default ztimer millisecond clock
 */
extern ztimer_clock_t *const ZTIMER_MSEC;

/**
 * @brief   Base ztimer for the millisecond clock
 */
extern ztimer_clock_t *const ZTIMER_MSEC_BASE;
```

or B):

```
/**
 * @brief   Default ztimer microsecond clock
 */
extern ztimer_clock_t *const ZTIMER_USEC;

/**
 * @brief   Default ztimer millisecond clock
 */
extern ztimer_clock_t *const ZTIMER_MSEC;

/* default ztimer virtual devices */
/**
 * @brief   Base ztimer for the microsecond clock
 */
extern ztimer_clock_t *const ZTIMER_USEC_BASE;

/**
 * @brief   Base ztimer for the millisecond clock
 */
extern ztimer_clock_t *const ZTIMER_MSEC_BASE;
```

Reply:
B! and maybe beef up docs that those are the clocks that the others are based on, if converted, or the same if not converted, and that the frequency of them are available as ZTIMER_X_FREQ in ztimer/config.h. I offer beers as bribe!